product_packaging_multi_barcode/README.rst

```diff
@@ -7,7 +7,7 @@ Multiple barcodes on product packagings
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-   !! source digest: sha256:cbad2b941a855012dc1379aaf1de51fedce967a4ccc05ec8e8c4923b2e2502a1
+   !! source digest: sha256:fe1979a76bb20948f63a3bd26d33f088a2ff8fafdf1197d1bf468ff1e3ffbe22
```

Comment:
Can't these pre-commit changes on `product_packaging_multi_barcode` be canceled?